android/build.gradle

```diff
@@ -72,6 +72,29 @@ allprojects {
     // Want this last so that we never end up with a stale cache
     mavenLocal()
   }
+
+  configurations.all {
+    // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
```

Comment:
Just to make sure, we are not supporting ejecting to expokit anymore?

Reply:
That is correct, we do not support ejecting to ExpoKit anymore. 🎉